gef-ui/templates/includes/application-preview/facilities.njk

```diff
@@ -23,11 +23,20 @@
 
 {% if facilities.data.length > 0 %}
   {% for item in facilities.data %}
-    <header class="is-flex is-justify-content-space-between">
```

Comment:
I think this was just copied and pasted from `application-details/facilities.njk`. The other file needs it because there is sometimes a link displayed in the header section. Whereas in this template there's only ever the `h3` inside, so we should be safe to remove it.